README.md

```diff
@@ -1,114 +1,89 @@
 # EasyGuideView
-Android app新手引导高亮提示，简单易用
 
-## 效果
+A HMOS library which provides guide highlighting tips, simple and easy to use
 
-<img src="https://github.com/smuyyh/EasyGuideView/blob/master/screenshot/1.png?raw=true" width=270/>
-<img src="https://github.com/smuyyh/EasyGuideView/blob/master/screenshot/2.png?raw=true" width=270/>
+## Source
+Inspired by [smuyyh/EasyGuideView](https://github.com/smuyyh/EasyGuideView)
 
-## 添加依赖
+## Feature
+A HMOS library which provides guide highlighting tips, simple and easy to use
 
+<img src="screenshots/Screenshot1.png" width="256">
+<img src="screenshots/Screenshot2.png" width="256">
+
+## Dependency
+1. For using easyguideview module in sample app, include the source code and add the below dependencies in entry/build.gradle to generate hap/support.har.
+```groovy
+	dependencies {
+		implementation project(':library')
```

Comment:
this library name will give an issue during maven release. Change the library module name to "easyguideview"